src/test/java/entityAPI/AddressAPI.java

```diff
@@ -0,0 +1,23 @@
+package entityAPI;
+
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import test.ConfigLoader;
+
+import static io.restassured.RestAssured.given;
+
+public class AddressAPI {
+    private static final ConfigLoader properties = ConfigLoader.getInstance();
+    private static final String baseURI = properties.getPropertyValue("baseURI");
+    private static final String ADDRESS_ENDPOINT = properties.getPropertyValue("ADDRESS_ENDPOINT");
+
+    public static Response getAddress(RequestSpecification request) {
```

Comment:
ideally, each api method should take an object of the corresponding class as a parameter. Address in this example